composer.json

```diff
@@ -21,29 +21,29 @@
         "doctrine/orm": "^2.16",
         "league/commonmark": "^2.1",
         "symfony/apache-pack": "^1.0",
-        "symfony/asset": "6.4.*",
-        "symfony/console": "6.4.*",
-        "symfony/dotenv": "6.4.*",
-        "symfony/expression-language": "6.4.*",
+        "symfony/asset": "^6.4",
+        "symfony/console": "^6.4",
+        "symfony/dotenv": "^6.4",
+        "symfony/expression-language": "^6.4",
         "symfony/flex": "^2",
-        "symfony/form": "6.4.*",
-        "symfony/framework-bundle": "6.4.*",
-        "symfony/html-sanitizer": "6.4.*",
-        "symfony/http-client": "6.4.*",
-        "symfony/intl": "6.4.*",
-        "symfony/mailer": "6.4.*",
+        "symfony/form": "^6.4",
+        "symfony/framework-bundle": "^6.4",
+        "symfony/html-sanitizer": "^6.4",
+        "symfony/http-client": "^6.4",
+        "symfony/intl": "^6.4",
+        "symfony/mailer": "^6.4",
         "symfony/monolog-bundle": "^3.7",
         "symfony/polyfill-intl-messageformatter": "^1.12",
-        "symfony/runtime": "6.4.*",
-        "symfony/security-bundle": "6.4.*",
+        "symfony/runtime": "^6.4",
+        "symfony/security-bundle": "^6.4",
         "symfony/stimulus-bundle": "^2.12",
-        "symfony/string": "6.4.*",
-        "symfony/translation": "6.4.*",
-        "symfony/twig-bundle": "6.4.*",
+        "symfony/string": "^6.4",
+        "symfony/translation": "^6.4",
+        "symfony/twig-bundle": "^6.4",
         "symfony/ux-live-component": "^2.6",
-        "symfony/validator": "6.4.*",
+        "symfony/validator": "^6.4",
         "symfony/webpack-encore-bundle": "^2.1",
-        "symfony/yaml": "6.4.*",
```

Comment:
As this demo wants to make minor version bumps explicit, should we really switch to the `.*` range to a semver range (which is only equivalent in this case because we know that we won't ever release 6.5) ?

Reply:
You are right. Let's revert to using `6.4.*` as before. Thanks.